client/offchain/Cargo.toml

```diff
@@ -19,7 +19,6 @@ hex = "0.4"
 fnv = "1.0.6"
 futures = "0.3.16"
 futures-timer = "3.0.2"
-log = "0.4.8"
```

Comment:
why move to `tracing` from `log`

Reply:
We already use tracing as the logging backend and the logging output looks much better. 

I especially like that you can pass "attributes" that are logged as well like `id` here:
```
tracing::debug!(target: LOG_TARGET, id = %request_id.0, "Finished writing body");
```